bookmarks.html

```diff
@@ -10,29 +10,26 @@
     <header>
       <h1>Quiz App</h1>
     </header>
-    <!-- I will use question-card as a class later to style it - What is better to use - or _ when I write a class?  -->
     <section class="question-card">
-      <div class="div_question-card">
+      <div class="question-card__button">
```

Comment:
```suggestion
      <div class="question-card__answer-container">
```
To be honest, I think this would be more fitting, especially because you're using a flexbox as part of the class. This element has a fair bit more features and functionality than just being a button.

If it were `__button` then it should semantically only include the button.

Reply:
a good fit, thanks